Error when running for the first time: no download URL specified for snapshot files in config

**Describe the bug**
We ran hornet and had this error: `no download URL specified for snapshot files in config`

**To Reproduce**
 - Install hornet using the apt package: `sudo apt install hornet`
 - Create a config.json with this content: `{"dashboard":{"bindAddress":"0.0.0.0:8081","auth":{"sessionTimeout":"72h","username":"***","passwordHash":"***","passwordSalt":"***"}}}`
 - Run with `hornet` command

The output of the command is above

**Expected behavior**
According to the [documentation](https://hornet.docs.iota.org/post_installation/managing.html#how-to-work-with-snapshots) they should be downloaded automatically from trusted source thanks to hornet, and we shouldn't have this error. 

**Environment information:**

 -  OS Debian 10
 - RAM: 4 GB
 - Cores: 4 Cores
 - Type:  VPS
 - HORNET version  1.0.3

**Additional context**

```
No profiles config file found via 'profiles.json'. Loading default settings.Parameters loaded: 
 {
  "coordinator": {
    "checkpoints": {
      "maxtrackedmessages": 10000
    },
    "interval": "10s",
    "powworkercount": 1,
    "quorum": {
      "enabled": false,
      "timeout": "2s"
    },
    "signing": {
      "provider": "local",
      "remoteaddress": "localhost:12345"
    },
    "statefilepath": "coordinator.state",
    "tipsel": {
      "heaviestbranchselectiontimeout": "100ms",
      "maxheaviestbranchtipspercheckpoint": 10,
      "minheaviestbranchunreferencedmessagesthreshold": 20,
      "randomtipspercheckpoint": 3
    }
  },
  "dashboard": {
    "auth": {
      "sessiontimeout": "72h",
      "username": "admin"
    },
    "bindaddress": "0.0.0.0:8081",
    "dev": false
  },
  "db": {
    "autorevalidation": false,
    "debug": false,
    "engine": "rocksdb",
    "path": "mainnetdb"
  },
  "debug": {
    "whiteflagparentssolidtimeout": "2s"
  },
  "gossip": {
    "streamreadtimeout": "1m0s",
    "streamwritetimeout": "10s"
  },
  "migrator": {
    "querycooldownperiod": "5s",
    "receiptmaxentries": 110,
    "statefilepath": "migrator.state"
  },
  "mqtt": {
    "bindaddress": "localhost:1883",
    "workercount": 100,
    "wsport": 1888
  },
  "node": {
    "alias": "",
    "disableplugins": [],
    "enableplugins": [],
    "profile": "auto"
  },
  "p2p": {
    "bindmultiaddresses": [
      "/ip4/127.0.0.1/tcp/15600"
    ],
    "connectionmanager": {
      "highwatermark": 10,
      "lowwatermark": 5
    },
    "gossipunknownpeerslimit": 4,
    "peerstore": {
      "path": "./p2pstore"
    },
    "reconnectinterval": "30s"
  },
  "p2pdisc": {
    "advertiseinterval": "30s",
    "maxdiscoveredpeerconns": 4,
    "rendezvouspoint": "between-two-vertices",
    "routingtablerefreshperiod": "1m0s"
  },
  "pow": {
    "refreshtipsinterval": "5s"
  },
  "profiling": {
    "bindaddress": "localhost:6060"
  },
  "prometheus": {
    "bindaddress": "localhost:9311",
    "cachesmetrics": true,
    "coordinatormetrics": true,
    "databasemetrics": true,
    "debugmetrics": false,
    "fileservicediscovery": {
      "enabled": false,
      "path": "target.json",
      "target": "localhost:9311"
    },
    "gometrics": false,
    "gossipmetrics": true,
    "migrationmetrics": true,
    "nodemetrics": true,
    "processmetrics": false,
    "promhttpmetrics": false,
    "restapimetrics": true
  },
  "protocol": {
    "bech32hrp": "iota",
    "milestonepublickeycount": 2,
    "minpowscore": "4000",
    "networkid": "iota"
  },
  "pruning": {
    "milestones": {
      "enabled": false,
      "maxmilestonestokeep": 60480
    },
    "prunereceipts": false,
    "size": {
      "cooldowntime": "5m0s",
      "enabled": true,
      "targetsize": "30GB",
      "thresholdpercentage": "10"
    }
  },
  "receipts": {
    "backup": {
      "enabled": false,
      "folder": "receipts"
    },
    "validator": {
      "api": {
        "address": "http://localhost:14266",
        "timeout": "5s"
      },
      "coordinator": {
        "address": "JFQ999DVN9CBBQX9DSAIQRAFRALIHJMYOXAQSTCJLGA9DLOKIWHJIFQKMCQ9QHWW9RXQMDBVUIQNIY9GZ",
        "merkletreedepth": 18
      },
      "ignoresofterrors": false,
      "validate": false
    }
  },
  "requests": {
    "discardolderthan": "15s",
    "pendingreenqueueinterval": "5s"
  },
  "restapi": {
    "bindaddress": "0.0.0.0:14265",
    "excludehealthcheckfromauth": false,
    "jwtauth": {
      "enabled": false
    },
    "limits": {
      "bodylength": "1M",
      "maxresults": 1000
    },
    "permittedroutes": [
      "/health",
      "/mqtt",
      "/api/v1/info",
      "/api/v1/tips",
      "/api/v1/messages/:messageID",
      "/api/v1/messages/:messageID/metadata",
      "/api/v1/messages/:messageID/raw",
      "/api/v1/messages/:messageID/children",
      "/api/v1/messages",
      "/api/v1/transactions/:transactionID/included-message",
      "/api/v1/milestones/:milestoneIndex",
      "/api/v1/milestones/:milestoneIndex/utxo-changes",
      "/api/v1/outputs/:outputID",
      "/api/v1/addresses/:address",
      "/api/v1/addresses/:address/outputs",
      "/api/v1/addresses/ed25519/:address",
      "/api/v1/addresses/ed25519/:address/outputs"
    ],
    "powenabled": false,
    "powworkercount": 1,
    "whitelistedaddresses": [
      "127.0.0.1",
      "::1"
    ]
  },
  "snapshots": {
    "deltapath": "snapshots/mainnet/delta_snapshot.bin",
    "deltasizethresholdpercentage": "50",
    "depth": 50,
    "fullpath": "snapshots/mainnet/full_snapshot.bin",
    "interval": 50
  },
  "spammer": {
    "autostart": false,
    "cpumaxusage": "0.5",
    "index": "HORNET Spammer",
    "indexsemilazy": "HORNET Spammer Semi-Lazy",
    "message": "Binary is the future.",
    "mpsratelimit": "0",
    "workers": 1
  },
  "tangle": {
    "milestonetimeout": "30s"
  },
  "tipsel": {
    "belowmaxdepth": 15,
    "maxdeltamsgoldestconerootindextocmi": 13,
    "maxdeltamsgyoungestconerootindextocmi": 8,
    "nonlazy": {
      "maxchildren": 30,
      "maxreferencedtipage": "3s",
      "retentionrulestipslimit": 100,
      "spammertipsthreshold": 0
    },
    "semilazy": {
      "maxchildren": 2,
      "maxreferencedtipage": "3s",
      "retentionrulestipslimit": 20,
      "spammertipsthreshold": 30
    }
  },
  "warpsync": {
    "advancementrange": 150
  }
}



              ██╗  ██╗ ██████╗ ██████╗ ███╗   ██╗███████╗████████╗
              ██║  ██║██╔═══██╗██╔══██╗████╗  ██║██╔════╝╚══██╔══╝
              ███████║██║   ██║██████╔╝██╔██╗ ██║█████╗     ██║
              ██╔══██║██║   ██║██╔══██╗██║╚██╗██║██╔══╝     ██║
              ██║  ██║╚██████╔╝██║  ██║██║ ╚████║███████╗   ██║
              ╚═╝  ╚═╝ ╚═════╝ ╚═╝  ╚═╝╚═╝  ╚═══╝╚══════╝   ╚═╝
                                   v1.0.3


2021-07-24T13:27:05Z	INFO	App	app/app.go:174	Loading plugins ...
2021-07-24T13:27:05Z	INFO	Profile	profile/core.go:62	Profile mode 'auto', Using profile '4gb'
2021-07-24T13:27:05Z	INFO	Node	node/node.go:214	Loading core plugin: Profile ... done
2021-07-24T13:27:05Z	INFO	Node	node/node.go:214	Loading core plugin: ProtoCfg ... done
2021-07-24T13:27:05Z	INFO	Node	node/node.go:214	Loading core plugin: Graceful Shutdown ... done
2021-07-24T13:27:05Z	INFO	Node	node/node.go:214	Loading core plugin: Database ... done
2021-07-24T13:27:05Z	INFO	Node	node/node.go:214	Loading core plugin: PoW ... done
2021-07-24T13:27:05Z	INFO	P2P	p2p/core.go:215	never share your ./p2pstore folder as it contains your node's private key!
2021-07-24T13:27:05Z	INFO	P2P	p2p/core.go:222	retrieved existing peer identity from p2pstore/key.pub
2021-07-24T13:27:05Z	INFO	P2P	p2p/core.go:225	peer configured, ID: *******
2021-07-24T13:27:05Z	INFO	Node	node/node.go:214	Loading core plugin: P2P ... done
2021-07-24T13:27:05Z	INFO	Node	node/node.go:214	Loading core plugin: Gossip ... done
2021-07-24T13:27:05Z	INFO	Node	node/node.go:214	Loading core plugin: Tangle ... done
2021-07-24T13:27:05Z	PANIC	Snapshot	snapshot/core.go:186	no download URL specified for snapshot files in config
github.com/gohornet/hornet/core/snapshot.configure
	/__w/hornet/hornet/core/snapshot/core.go:186
github.com/gohornet/hornet/pkg/node.(*Node).configure.func1
	/__w/hornet/hornet/pkg/node/node.go:212
github.com/gohornet/hornet/pkg/node.forEachCorePlugin
	/__w/hornet/hornet/pkg/node/node.go:320
github.com/gohornet/hornet/pkg/node.(*Node).ForEachCorePlugin
	/__w/hornet/hornet/pkg/node/node.go:328
github.com/gohornet/hornet/pkg/node.(*Node).configure
	/__w/hornet/hornet/pkg/node/node.go:210
github.com/gohornet/hornet/pkg/node.New
	/__w/hornet/hornet/pkg/node/node.go:49
github.com/gohornet/hornet/pkg/node.Run
	/__w/hornet/hornet/pkg/node/node.go:62
main.main
	/__w/hornet/hornet/main.go:33
runtime.main
	/usr/local/go/src/runtime/proc.go:225
panic: no download URL specified for snapshot files in config

goroutine 1 [running]:
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc0008d0300, 0x0, 0x0, 0x0)
	/github/home/go/pkg/mod/go.uber.org/zap@v1.16.0/zapcore/entry.go:234 +0x58d
go.uber.org/zap.(*SugaredLogger).log(0xc00011f4c0, 0x4, 0x0, 0x0, 0xc000c59d30, 0x1, 0x1, 0x0, 0x0, 0x0)
	/github/home/go/pkg/mod/go.uber.org/zap@v1.16.0/sugar.go:234 +0xf6
go.uber.org/zap.(*SugaredLogger).Panic(...)
	/github/home/go/pkg/mod/go.uber.org/zap@v1.16.0/sugar.go:123
github.com/gohornet/hornet/core/snapshot.configure()
	/__w/hornet/hornet/core/snapshot/core.go:186 +0x119
github.com/gohornet/hornet/pkg/node.(*Node).configure.func1(0xc0001305f0, 0x1b70f01)
	/__w/hornet/hornet/pkg/node/node.go:212 +0xdd
github.com/gohornet/hornet/pkg/node.forEachCorePlugin(0xc0000c3e00, 0x9, 0x10, 0xc000c59e40)
	/__w/hornet/hornet/pkg/node/node.go:320 +0x51
github.com/gohornet/hornet/pkg/node.(*Node).ForEachCorePlugin(...)
	/__w/hornet/hornet/pkg/node/node.go:328
github.com/gohornet/hornet/pkg/node.(*Node).configure(0xc0004128c0)
	/__w/hornet/hornet/pkg/node/node.go:210 +0x7c
github.com/gohornet/hornet/pkg/node.New(0xc00089ff60, 0x3, 0x3, 0x1c58060)
	/__w/hornet/hornet/pkg/node/node.go:49 +0x248
github.com/gohornet/hornet/pkg/node.Run(0xc00089ff60, 0x3, 0x3, 0xc000281da0)
	/__w/hornet/hornet/pkg/node/node.go:62 +0x3f
main.main()
	/__w/hornet/hornet/main.go:33 +0x334
```

--- a/core/app/app.go
+++ b/core/app/app.go
@@ -103,6 +103,10 @@ Command line flags:
 		return nil, err
 	}
 
+	if err = nodeConfig.SetDefault(logger.ConfigurationKeyDisableCaller, true); err != nil {
+		panic(err)
+	}
+
 	if err = logger.InitGlobalLogger(nodeConfig); err != nil {
 		panic(err)
 	}

--- a/core/p2p/params.go
+++ b/core/p2p/params.go
@@ -9,35 +9,35 @@ import (
 )
 
 const (
-	// Defines the private key used to derive the node identity (optional).
-	CfgP2PIdentityPrivKey = "p2p.identityPrivateKey"
 	// Defines the bind addresses of this node.
 	CfgP2PBindMultiAddresses = "p2p.bindMultiAddresses"
-	// Defines the path to the peerstore.
-	CfgP2PPeerStorePath = "p2p.peerStore.path"
 	// Defines the high watermark to use within the connection manager.
 	CfgP2PConnMngHighWatermark = "p2p.connectionManager.highWatermark"
 	// Defines the low watermark to use within the connection manager.
 	CfgP2PConnMngLowWatermark = "p2p.connectionManager.lowWatermark"
+	// Defines the private key used to derive the node identity (optional).
+	CfgP2PIdentityPrivKey = "p2p.identityPrivateKey"
+	// Defines the path to the peerstore.
+	CfgP2PPeerStorePath = "p2p.peerStore.path"
+	// Defines the time to wait before trying to reconnect to a disconnected peer.
+	CfgP2PReconnectInterval = "p2p.reconnectInterval"
 	// Defines the static peers this node should retain a connection to (config file).
-	CfgPeers = "peers"
-	// Defines the static peers this node should retain a connection to (CLI).
 	CfgP2PPeers = "p2p.peers"
 	// Defines the aliases of the static peers (must be the same length like CfgP2PPeers) (CLI).
 	CfgP2PPeerAliases = "p2p.peerAliases"
-	// Defines the time to wait before trying to reconnect to a disconnected peer.
-	CfgP2PReconnectInterval = "p2p.reconnectInterval"
+	// Defines the static peers this node should retain a connection to (CLI).
+	CfgPeers = "peers"
 )
 
 var params = &node.PluginParams{
 	Params: map[string]*flag.FlagSet{
 		"nodeConfig": func() *flag.FlagSet {
 			fs := flag.NewFlagSet("", flag.ContinueOnError)
-			fs.String(CfgP2PIdentityPrivKey, "", "private key used to derive the node identity (optional)")
 			fs.StringSlice(CfgP2PBindMultiAddresses, []string{"/ip4/127.0.0.1/tcp/15600"}, "the bind addresses for this node")
-			fs.String(CfgP2PPeerStorePath, "./p2pstore", "the path to the peer store")
 			fs.Int(CfgP2PConnMngHighWatermark, 10, "the threshold up on which connections count truncates to the lower watermark")
 			fs.Int(CfgP2PConnMngLowWatermark, 5, "the minimum connections count to hold after the high watermark was reached")
+			fs.String(CfgP2PIdentityPrivKey, "", "private key used to derive the node identity (optional)")
+			fs.String(CfgP2PPeerStorePath, "./p2pstore", "the path to the peer store")
 			fs.Duration(CfgP2PReconnectInterval, 30*time.Second, "the time to wait before trying to reconnect to a disconnected peer")
 			return fs
 		}(),

--- a/core/protocfg/core.go
+++ b/core/protocfg/core.go
@@ -13,18 +13,18 @@ import (
 )
 
 const (
-	// the ed25519 public key of the coordinator in hex representation
-	CfgProtocolPublicKeyRangesJSON = "publicKeyRanges"
-	// the ed25519 public key of the coordinator in hex representation
-	CfgProtocolPublicKeyRanges = "protocol.publicKeyRanges"
-	// the minimum PoW score required by the network
-	CfgProtocolMinPoWScore = "protocol.minPoWScore"
 	// the network ID on which this node operates on.
 	CfgProtocolNetworkIDName = "protocol.networkID"
 	// the HRP which should be used for Bech32 addresses.
 	CfgProtocolBech32HRP = "protocol.bech32HRP"
-	// the amount of public keys in a milestone
+	// the minimum PoW score required by the network.
+	CfgProtocolMinPoWScore = "protocol.minPoWScore"
+	// the amount of public keys in a milestone.
 	CfgProtocolMilestonePublicKeyCount = "protocol.milestonePublicKeyCount"
+	// the ed25519 public key of the coordinator in hex representation.
+	CfgProtocolPublicKeyRanges = "protocol.publicKeyRanges"
+	// the ed25519 public key of the coordinator in hex representation.
+	CfgProtocolPublicKeyRangesJSON = "publicKeyRanges"
 )
 
 func init() {
@@ -37,10 +37,10 @@ func init() {
 				Params: map[string]*flag.FlagSet{
 					"nodeConfig": func() *flag.FlagSet {
 						fs := flag.NewFlagSet("", flag.ContinueOnError)
+						fs.String(CfgProtocolNetworkIDName, "chrysalis-mainnet", "the network ID on which this node operates on.")
+						fs.String(CfgProtocolBech32HRP, string(iotago.PrefixMainnet), "the HRP which should be used for Bech32 addresses.")
 						fs.Float64(CfgProtocolMinPoWScore, 4000, "the minimum PoW score required by the network.")
 						fs.Int(CfgProtocolMilestonePublicKeyCount, 2, "the amount of public keys in a milestone")
-						fs.String(CfgProtocolNetworkIDName, "iota", "the network ID on which this node operates on.")
-						fs.String(CfgProtocolBech32HRP, string(iotago.PrefixMainnet), "the HRP which should be used for Bech32 addresses.")
 						return fs
 					}(),
 				},
@@ -86,6 +86,24 @@ func provide(c *dig.Container) {
 				CorePlugin.Panic(err)
 			}
 			return res
+		}
+
+		if err := deps.NodeConfig.SetDefault(CfgProtocolPublicKeyRanges, coordinator.PublicKeyRanges{
+			{
+				Key:        "a9b46fe743df783dedd00c954612428b34241f5913cf249d75bed3aafd65e4cd",
+				StartIndex: 0,
+				EndIndex:   777600,
+			}, {
+				Key:        "365fb85e7568b9b32f7359d6cbafa9814472ad0ecbad32d77beaf5dd9e84c6ba",
+				StartIndex: 0,
+				EndIndex:   1555200,
+			}, {
+				Key:        "ba6d07d1a1aea969e7e435f9f7d1b736ea9e0fcb8de400bf855dba7f2a57e947",
+				StartIndex: 552960,
+				EndIndex:   2108160,
+			},
+		}); err != nil {
+			CorePlugin.Panic(err)
 		}
 
 		// load from config

--- a/core/snapshot/core.go
+++ b/core/snapshot/core.go
@@ -87,7 +87,15 @@ func provide(c *dig.Container) {
 
 		networkIDSource := deps.NodeConfig.String(protocfg.CfgProtocolNetworkIDName)
 
-		if err := deps.NodeConfig.SetDefault(CfgSnapshotsDownloadURLs, []snapshot.DownloadTarget{}); err != nil {
+		if err := deps.NodeConfig.SetDefault(CfgSnapshotsDownloadURLs, []snapshot.DownloadTarget{
+			{
+				Full:  "https://chrysalis-dbfiles.iota.org/snapshots/hornet/latest-full_snapshot.bin",
+				Delta: "https://chrysalis-dbfiles.iota.org/snapshots/hornet/latest-delta_snapshot.bin",
+			},
+			{
+				Full: "https://mainnet.tanglebay.com/ls/full_snapshot.bin",
+			},
+		}); err != nil {
 			CorePlugin.Panic(err)
 		}
 

--- a/documentation/docs/post_installation/managing_a_node.md
+++ b/documentation/docs/post_installation/managing_a_node.md
@@ -24,16 +24,11 @@ Another important directory is the `snapshots` directory.  You can control the `
     "deltaSizeThresholdPercentage": 50.0,
     "downloadURLs": [
       {
-        "full": "https://ls.manapotion.io/full_snapshot.bin",
-        "delta": "https://ls.manapotion.io/delta_snapshot.bin"
+        "full": "https://chrysalis-dbfiles.iota.org/snapshots/hornet/latest-full_snapshot.bin",
+        "delta": "https://chrysalis-dbfiles.iota.org/snapshots/hornet/latest-delta_snapshot.bin"
       },
       {
-        "full": "https://x-vps.com/full_snapshot.bin",
-        "delta": "https://x-vps.com/delta_snapshot.bin"
-      },
-      {
-        "full": "https://dbfiles.iota.org/mainnet/hornet/full_snapshot.bin",
-        "delta": "https://dbfiles.iota.org/mainnet/hornet/delta_snapshot.bin"
+        "full": "https://mainnet.tanglebay.com/ls/full_snapshot.bin"
       }
     ]
 ```
@@ -105,16 +100,11 @@ Your node's ledger accumulates many messages, which uses a significant disk capa
     "deltaSizeThresholdPercentage": 50.0,
     "downloadURLs": [
       {
-        "full": "https://ls.manapotion.io/full_snapshot.bin",
-        "delta": "https://ls.manapotion.io/delta_snapshot.bin"
+        "full": "https://chrysalis-dbfiles.iota.org/snapshots/hornet/latest-full_snapshot.bin",
+        "delta": "https://chrysalis-dbfiles.iota.org/snapshots/hornet/latest-delta_snapshot.bin"
       },
       {
-        "full": "https://x-vps.com/full_snapshot.bin",
-        "delta": "https://x-vps.com/delta_snapshot.bin"
-      },
-      {
-        "full": "https://dbfiles.iota.org/mainnet/hornet/full_snapshot.bin",
-        "delta": "https://dbfiles.iota.org/mainnet/hornet/delta_snapshot.bin"
+        "full": "https://mainnet.tanglebay.com/ls/full_snapshot.bin"
       }
     ]
   },

--- a/pkg/toolset/jwt.go
+++ b/pkg/toolset/jwt.go
@@ -16,9 +16,9 @@ func generateJWTApiToken(nodeConfig *configuration.Configuration, args []string)
 		return fmt.Errorf("too many arguments for '%s'", ToolJWTApi)
 	}
 
-	salt := nodeConfig.String(restapi.CfgRestAPIAuthSalt)
+	salt := nodeConfig.String(restapi.CfgRestAPIJWTAuthSalt)
 	if len(salt) == 0 {
-		return fmt.Errorf("'%s' should not be empty", restapi.CfgRestAPIAuthSalt)
+		return fmt.Errorf("'%s' should not be empty", restapi.CfgRestAPIJWTAuthSalt)
 	}
 
 	prvKey, pid, err := loadP2PPrivKeyAndIdentityFromStore(nodeConfig.String(p2p.CfgP2PPeerStorePath))

--- a/plugins/autopeering/params.go
+++ b/plugins/autopeering/params.go
@@ -9,10 +9,14 @@ import (
 )
 
 const (
-	// CfgNetAutopeeringEntryNodes list of autopeering entry nodes to use.
-	CfgNetAutopeeringEntryNodes = "p2p.autopeering.entryNodes"
 	// CfgNetAutopeeringBindAddr is bind address for autopeering.
 	CfgNetAutopeeringBindAddr = "p2p.autopeering.bindAddress"
+	// CfgNetAutopeeringDatabasePath is the path to the autopeering database.
+	CfgNetAutopeeringDatabasePath = "p2p.autopeering.db.path"
+	// CfgNetAutopeeringEntryNodes list of autopeering entry nodes to use.
+	CfgNetAutopeeringEntryNodes = "p2p.autopeering.entryNodes"
+	// CfgNetAutopeeringEntryNodesPreferIPv6 defines if connecting over IPv6 is preferred for entry nodes.
+	CfgNetAutopeeringEntryNodesPreferIPv6 = "p2p.autopeering.entryNodesPreferIPv6"
 	// CfgNetAutopeeringRunAsEntryNode whether the node should act as an autopeering entry node.
 	CfgNetAutopeeringRunAsEntryNode = "p2p.autopeering.runAsEntryNode"
 	// CfgNetAutopeeringInboundPeers the number of inbound autopeers.
@@ -21,24 +25,20 @@ const (
 	CfgNetAutopeeringOutboundPeers = "p2p.autopeering.outboundPeers"
 	// CfgNetAutopeeringSaltLifetime lifetime of the private and public local salt.
 	CfgNetAutopeeringSaltLifetime = "p2p.autopeering.saltLifetime"
-	// CfgNetAutopeeringDatabasePath is the path to the autopeering database.
-	CfgNetAutopeeringDatabasePath = "p2p.autopeering.db.path"
-	// CfgNetAutopeeringEntryNodesPreferIPv6 defines if connecting over IPv6 is preferred for entry nodes.
-	CfgNetAutopeeringEntryNodesPreferIPv6 = "p2p.autopeering.entryNodesPreferIPv6"
 )
 
 var params = &node.PluginParams{
 	Params: map[string]*flag.FlagSet{
 		"nodeConfig": func() *flag.FlagSet {
 			fs := flag.NewFlagSet("", flag.ContinueOnError)
-			fs.StringSlice(CfgNetAutopeeringEntryNodes, []string{}, "list of autopeering entry nodes to use")
 			fs.String(CfgNetAutopeeringBindAddr, "0.0.0.0:14626", "bind address for autopeering")
-			fs.Bool(CfgNetAutopeeringRunAsEntryNode, false, "whether the node should act as an autopeering entry node")
+			fs.String(CfgNetAutopeeringDatabasePath, "./p2pstore", "the directory path (excluding the name) to the autopeering database")
+			fs.StringSlice(CfgNetAutopeeringEntryNodes, []string{}, "list of autopeering entry nodes to use")
 			fs.Bool(CfgNetAutopeeringEntryNodesPreferIPv6, false, "defines if connecting over IPv6 is preferred for entry nodes")
+			fs.Bool(CfgNetAutopeeringRunAsEntryNode, false, "whether the node should act as an autopeering entry node")
 			fs.Int(CfgNetAutopeeringInboundPeers, 2, "the number of inbound autopeers")
 			fs.Int(CfgNetAutopeeringOutboundPeers, 2, "the number of outbound autopeers")
 			fs.Duration(CfgNetAutopeeringSaltLifetime, 2*time.Hour, "lifetime of the private and public local salt")
-			fs.String(CfgNetAutopeeringDatabasePath, "./p2pstore", "the directory path (excluding the name) to the autopeering database")
 			return fs
 		}(),
 	},

--- a/plugins/dashboard/params.go
+++ b/plugins/dashboard/params.go
@@ -29,7 +29,7 @@ var params = &node.PluginParams{
 	Params: map[string]*flag.FlagSet{
 		"nodeConfig": func() *flag.FlagSet {
 			fs := flag.NewFlagSet("", flag.ContinueOnError)
-			fs.String(CfgNodeAlias, "", "set an alias to identify a node")
+			fs.String(CfgNodeAlias, "HORNET node", "set an alias to identify a node")
 			fs.String(CfgDashboardBindAddress, "localhost:8081", "the bind address on which the dashboard can be access from")
 			fs.Bool(CfgDashboardDevMode, false, "whether to run the dashboard in dev mode")
 			fs.Duration(CfgDashboardAuthSessionTimeout, 72*time.Hour, "how long the auth session should last before expiring")

--- a/plugins/prometheus/debug.go
+++ b/plugins/prometheus/debug.go
@@ -87,21 +87,21 @@ func configureDebug() {
 	)
 
 	deps.Snapshot.Events.SnapshotMetricsUpdated.Attach(events.NewClosure(func(metrics *snapshot.SnapshotMetrics) {
-		snapshotTotalDuration.Observe(float64(metrics.DurationTotal.Seconds()))
+		snapshotTotalDuration.Observe(metrics.DurationTotal.Seconds())
 		metricsLock.Lock()
 		defer metricsLock.Unlock()
 		lastSnapshotMetrics = metrics
 	}))
 
 	deps.Snapshot.Events.PruningMetricsUpdated.Attach(events.NewClosure(func(metrics *snapshot.PruningMetrics) {
-		databasePruningTotalDuration.Observe(float64(metrics.DurationTotal.Seconds()))
+		databasePruningTotalDuration.Observe(metrics.DurationTotal.Seconds())
 		metricsLock.Lock()
 		defer metricsLock.Unlock()
 		lastDatabasePruningMetrics = metrics
 	}))
 
 	deps.Tangle.Events.ConfirmationMetricsUpdated.Attach(events.NewClosure(func(metrics *whiteflag.ConfirmationMetrics) {
-		milestoneConfirmationTotalDuration.Observe(float64(metrics.DurationTotal.Seconds()))
+		milestoneConfirmationTotalDuration.Observe(metrics.DurationTotal.Seconds())
 		metricsLock.Lock()
 		defer metricsLock.Unlock()
 		lastConfirmationMetrics = metrics

--- a/plugins/prometheus/params.go
+++ b/plugins/prometheus/params.go
@@ -7,36 +7,36 @@ import (
 )
 
 const (
-	// the bind address on which the Prometheus exporter listens on
+	// the bind address on which the Prometheus exporter listens on.
 	CfgPrometheusBindAddress = "prometheus.bindAddress"
-	// include database metrics
-	CfgPrometheusDatabase = "prometheus.databaseMetrics"
-	// include node metrics
-	CfgPrometheusNode = "prometheus.nodeMetrics"
-	// include gossip metrics
-	CfgPrometheusGossip = "prometheus.gossipMetrics"
-	// include caches metrics
-	CfgPrometheusCaches = "prometheus.cachesMetrics"
-	// include restAPI metrics
-	CfgPrometheusRestAPI = "prometheus.restAPIMetrics"
-	// include migration metrics
-	CfgPrometheusMigration = "prometheus.migrationMetrics"
-	// include coordinator metrics
-	CfgPrometheusCoordinator = "prometheus.coordinatorMetrics"
-	// include debug metrics
-	CfgPrometheusDebug = "prometheus.debugMetrics"
-	// include go metrics
-	CfgPrometheusGoMetrics = "prometheus.goMetrics"
-	// include process metrics
-	CfgPrometheusProcessMetrics = "prometheus.processMetrics"
-	// include promhttp metrics
-	CfgPrometheusPromhttpMetrics = "prometheus.promhttpMetrics"
-	// whether the plugin should write a Prometheus 'file SD' file
+	// whether the plugin should write a Prometheus 'file SD' file.
 	CfgPrometheusFileServiceDiscoveryEnabled = "prometheus.fileServiceDiscovery.enabled"
-	// the path where to write the 'file SD' file to
+	// the path where to write the 'file SD' file to.
 	CfgPrometheusFileServiceDiscoveryPath = "prometheus.fileServiceDiscovery.path"
-	// the target to write into the 'file SD' file
+	// the target to write into the 'file SD' file.
 	CfgPrometheusFileServiceDiscoveryTarget = "prometheus.fileServiceDiscovery.target"
+	// include database metrics.
+	CfgPrometheusDatabase = "prometheus.databaseMetrics"
+	// include node metrics.
+	CfgPrometheusNode = "prometheus.nodeMetrics"
+	// include gossip metrics.
+	CfgPrometheusGossip = "prometheus.gossipMetrics"
+	// include caches metrics.
+	CfgPrometheusCaches = "prometheus.cachesMetrics"
+	// include restAPI metrics.
+	CfgPrometheusRestAPI = "prometheus.restAPIMetrics"
+	// include migration metrics.
+	CfgPrometheusMigration = "prometheus.migrationMetrics"
+	// include coordinator metrics.
+	CfgPrometheusCoordinator = "prometheus.coordinatorMetrics"
+	// include debug metrics.
+	CfgPrometheusDebug = "prometheus.debugMetrics"
+	// include go metrics.
+	CfgPrometheusGoMetrics = "prometheus.goMetrics"
+	// include process metrics.
+	CfgPrometheusProcessMetrics = "prometheus.processMetrics"
+	// include promhttp metrics.
+	CfgPrometheusPromhttpMetrics = "prometheus.promhttpMetrics"
 )
 
 var params = &node.PluginParams{
@@ -44,6 +44,9 @@ var params = &node.PluginParams{
 		"nodeConfig": func() *flag.FlagSet {
 			fs := flag.NewFlagSet("", flag.ContinueOnError)
 			fs.String(CfgPrometheusBindAddress, "localhost:9311", "the bind address on which the Prometheus exporter listens on")
+			fs.Bool(CfgPrometheusFileServiceDiscoveryEnabled, false, "whether the plugin should write a Prometheus 'file SD' file")
+			fs.String(CfgPrometheusFileServiceDiscoveryPath, "target.json", "the path where to write the 'file SD' file to")
+			fs.String(CfgPrometheusFileServiceDiscoveryTarget, "localhost:9311", "the target to write into the 'file SD' file")
 			fs.Bool(CfgPrometheusDatabase, true, "include database metrics")
 			fs.Bool(CfgPrometheusNode, true, "include node metrics")
 			fs.Bool(CfgPrometheusGossip, true, "include gossip metrics")
@@ -55,9 +58,6 @@ var params = &node.PluginParams{
 			fs.Bool(CfgPrometheusGoMetrics, false, "include go metrics")
 			fs.Bool(CfgPrometheusProcessMetrics, false, "include process metrics")
 			fs.Bool(CfgPrometheusPromhttpMetrics, false, "include promhttp metrics")
-			fs.Bool(CfgPrometheusFileServiceDiscoveryEnabled, false, "whether the plugin should write a Prometheus 'file SD' file")
-			fs.String(CfgPrometheusFileServiceDiscoveryPath, "target.json", "the path where to write the 'file SD' file to")
-			fs.String(CfgPrometheusFileServiceDiscoveryTarget, "localhost:9311", "the target to write into the 'file SD' file")
 			return fs
 		}(),
 	},

--- a/plugins/receipt/param.go
+++ b/plugins/receipt/param.go
@@ -31,10 +31,10 @@ var params = &node.PluginParams{
 	Params: map[string]*flag.FlagSet{
 		"nodeConfig": func() *flag.FlagSet {
 			fs := flag.NewFlagSet("", flag.ContinueOnError)
-			fs.String(CfgReceiptsBackupPath, "receipts", "path to the receipts backup folder")
 			fs.Bool(CfgReceiptsBackupEnabled, false, "whether to backup receipts in the backup folder")
-			fs.Bool(CfgReceiptsValidatorIgnoreSoftErrors, false, "whether to ignore soft errors and not panic if one is encountered")
+			fs.String(CfgReceiptsBackupPath, "receipts", "path to the receipts backup folder")
 			fs.Bool(CfgReceiptsValidatorValidate, false, "whether to validate receipts")
+			fs.Bool(CfgReceiptsValidatorIgnoreSoftErrors, false, "whether to ignore soft errors and not panic if one is encountered")
 			fs.String(CfgReceiptsValidatorAPIAddress, "http://localhost:14266", "address of the legacy node API")
 			fs.Duration(CfgReceiptsValidatorAPITimeout, 5*time.Second, "timeout of API calls")
 			fs.String(CfgReceiptsValidatorCoordinatorAddress, "UDYXTZBE9GZGPM9SSQV9LTZNDLJIZMPUVVXYXFYVBLIEUHLSEWFTKZZLXYRHHWVQV9MNNX9KZC9D9UZWZ", "address of the legacy coordinator")

--- a/plugins/restapi/params.go
+++ b/plugins/restapi/params.go
@@ -16,9 +16,9 @@ const (
 	// whether to allow the health check route anyways
 	CfgRestAPIExcludeHealthCheckFromAuth = "restAPI.excludeHealthCheckFromAuth"
 	// whether to use JWT auth for the REST API
-	CfgRestAPIAuthEnabled = "restAPI.jwtAuth.enabled"
+	CfgRestAPIJWTAuthEnabled = "restAPI.jwtAuth.enabled"
 	// salt used inside the JWT tokens for the REST API. Change this to a different value to invalidate JWT tokens not matching this new value
-	CfgRestAPIAuthSalt = "restAPI.jwtAuth.salt"
+	CfgRestAPIJWTAuthSalt = "restAPI.jwtAuth.salt"
 	// whether the node does PoW if messages are received via API
 	CfgRestAPIPoWEnabled = "restAPI.powEnabled"
 	// the amount of workers used for calculating PoW when issuing messages via API
@@ -57,8 +57,8 @@ var params = &node.PluginParams{
 				}, "the allowed HTTP REST routes which can be called from non whitelisted addresses")
 			fs.StringSlice(CfgRestAPIWhitelistedAddresses, []string{"127.0.0.1", "::1"}, "the whitelist of addresses which are allowed to access the REST API")
 			fs.Bool(CfgRestAPIExcludeHealthCheckFromAuth, false, "whether to allow the health check route anyways")
-			fs.Bool(CfgRestAPIAuthEnabled, false, "whether to use JWT auth for the REST API")
-			fs.String(CfgRestAPIAuthSalt, "HORNET", "salt used inside the JWT tokens for the REST API. Change this to a different value to invalidate JWT tokens not matching this new value")
+			fs.Bool(CfgRestAPIJWTAuthEnabled, false, "whether to use JWT auth for the REST API")
+			fs.String(CfgRestAPIJWTAuthSalt, "HORNET", "salt used inside the JWT tokens for the REST API. Change this to a different value to invalidate JWT tokens not matching this new value")
 			fs.Bool(CfgRestAPIPoWEnabled, false, "whether the node does PoW if messages are received via API")
 			fs.Int(CfgRestAPIPoWWorkerCount, 1, "the amount of workers used for calculating PoW when issuing messages via API")
 			fs.String(CfgRestAPILimitsMaxBodyLength, "1M", "the maximum number of characters that the body of an API call may contain")
@@ -66,5 +66,5 @@ var params = &node.PluginParams{
 			return fs
 		}(),
 	},
-	Masked: []string{CfgRestAPIAuthSalt},
+	Masked: []string{CfgRestAPIJWTAuthSalt},
 }

--- a/plugins/restapi/plugin.go
+++ b/plugins/restapi/plugin.go
@@ -115,11 +115,11 @@ func configure() {
 	deps.Echo.Use(middlewareFilterRoutes(whitelistedNetworks, permittedRoutes))
 
 	// set basic auth if enabled
-	if deps.NodeConfig.Bool(CfgRestAPIAuthEnabled) {
+	if deps.NodeConfig.Bool(CfgRestAPIJWTAuthEnabled) {
 
-		salt := deps.NodeConfig.String(CfgRestAPIAuthSalt)
+		salt := deps.NodeConfig.String(CfgRestAPIJWTAuthSalt)
 		if len(salt) == 0 {
-			Plugin.LogFatalf("'%s' should not be empty", CfgRestAPIAuthSalt)
+			Plugin.LogFatalf("'%s' should not be empty", CfgRestAPIJWTAuthSalt)
 		}
 
 		// API tokens do not expire.

